selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -67,11 +77,38 @@ def select_role_type(self, canvas_role):
         self.find_element(*Locators.ROLES_DROPDOWN_LIST).click()
         self.find_element(By.LINK_TEXT, canvas_role).click()
 
-    def add_was_successful(self):
+    def single_add_was_successful(self):
         # Verify success text
-        # todo: this does not check _which_ add was successful...
+        # todo: we can refactor this; see "add_is_successful_by_id" method
+        try:
+            self.find_element(*Locators.SINGLE_ADD_ALERT_TEXT)
+        except NoSuchElementException:
+            return False
+        return True
+
+
+    def add_is_successful_by_id(self, user_id):
+        # Verify user ID is displayed
+        try:
+            self.find_element(*CourseInfoBasePageObjectLocators.TD_TEXT_XPATH(
+                        user_id)).is_displayed()
+        except NoSuchElementException:
+            return False
+        return True
+
+    def multiple_add_partial_failure(self):
```

Comment:
Sorry, I meant `multiple_add_partial_failure()` (actually, that method name doesn't appear to be accurate anyway?) and `single_add_was_successful()`. They are both checking the message, but differ slightly -- it might be clearer if the service was something like `num_of_people_added_successfully(x)`, and in that method if `x==1` we check for the single add success message, if `x>1` we check for the multiple add success message.